fw-child/functions.php

```diff
@@ -430,7 +430,6 @@ function add_favicon() {
 <script async defer crossorigin="anonymous" src="https://connect.facebook.net/en_US/sdk.js"></script>
 
 <script src="https://platform.linkedin.com/in.js" type="text/javascript">lang: en_US</script>
-<script type="IN/Share" data-url="https://www.linkedin.com"></script>
```

Comment:
The button is removed, but then the included scripts above are included needlessly. It's probably the whole `if` that needs to be removed.